components/admin_console/team_channel_settings/group/group_users/group_users_groups.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Tooltip} from 'react-bootstrap';
+import {FormattedMessage} from 'react-intl';
+
+import {UserProfile} from 'mattermost-redux/types/users';
+import {Group} from 'mattermost-redux/types/groups';
+
+import {t} from 'utils/i18n';
+import OverlayTrigger from 'components/overlay_trigger';
+
+type ProfileWithGroups = Partial<UserProfile & {
+    groups: Partial<Group>[];
+}>;
+
+interface GroupUsersGroupsProps {
+    user: ProfileWithGroups;
+}
+export default class GroupUsersGroups extends React.PureComponent<GroupUsersGroupsProps, {}> {
+    renderGroupsColumn = (user: ProfileWithGroups) => {
+        const groups = user.groups || [];
+        if ((groups).length === 1) {
+            return groups[0].display_name;
+        }
+
+        const message = (
+            <FormattedMessage
+                id={t('team_channel_settings.group.group_user_row.numberOfGroups')}
```

Comment:
We shouldn't need to use `t` here. The tooling automatically recognizes translation strings inside of `FormattedMessage`s even without it.